scripts/should_rebuild_master.py

```diff
@@ -33,6 +33,10 @@ def should_rebuild_project(changed_files, project):
         print("*** Changes to the scripts directory mean we should rebuild")
         return True
 
+    elif any(f.startswith('.travis/') for f in changed_files):
```

Comment:
Yup. I agree, as this doesn't regress functionality I would merge and then or another change.